build/script/CIRestore.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
Was this a bad merge that this script still exists?

Reply:
I initially created this script as part of supporting richnav - it's no longer needed. Overall this PR is just for testing purposes as of now to see if this feature could work. I'm in contact with the team to see what work's been done on their side

Reply:
Do you need a concept of restore only?